openmetadata-spec/src/main/resources/json/schema/entity/services/connections/database/salesforceConnection.json

```diff
@@ -75,14 +64,6 @@
     "supportsMetadataExtraction": {
       "title": "Supports Metadata Extraction",
       "$ref": "../connectionBasicType.json#/definitions/supportsMetadataExtraction"
-    },
-    "supportsProfiler": {
-      "title": "Supports Profiler",
-      "$ref": "../connectionBasicType.json#/definitions/supportsProfiler"
-    },
-    "supportsQueryComment": {
-      "title": "Supports Query Comment",
```

Comment:
we do not execute any query to fetch data from salesforce hence this field is useless